repos/runtime.common.targets

```diff
@@ -22,6 +22,28 @@
                          Rid="$(TargetRid)-$(Platform)" />
   </Target>
 
+  <Target Name="CopyTools"
+          AfterTargets="Package">
```

Comment:
Why was this moved from `runtime-portable.proj` to `runtime.common.targets`? Does this mean both builds copy it? Does the last one win, or are they disambiguated in some way? I think we do want the portable one so our previously-source-built artifacts run on as many platforms as possible.

Reply:
Yes.  Both builds copy it.  The last one wins, which is portable if it is run.  There were cases where we wanted these copied when the portable build didn't run.